src/query/sql/src/planner/binder/ddl/table.rs

```diff
@@ -527,12 +529,20 @@ impl Binder {
         }
 
         // Build table schema
-        let (schema, field_comments, inverted_indexes, as_query_plan) = match (&source, &as_query) {
+        let (schema, field_comments, inverted_indexes, ngram_indexes, as_query_plan) = match (
+            &source, &as_query,
+        ) {
             (Some(source), None) => {
                 // `CREATE TABLE` without `AS SELECT ...`
-                let (schema, field_comments, inverted_indexes) =
+                let (schema, field_comments, inverted_indexes, ngram_indexes) =
                     self.analyze_create_table_schema(source).await?;
-                (schema, field_comments, inverted_indexes, None)
+                (
+                    schema,
+                    field_comments,
+                    inverted_indexes,
+                    ngram_indexes,
```

Comment:
need to check if the name of the inveted index and the ngram index are duplicated.